CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.3.1] - 2023-08-02
+
+### Changed
+* Default Check and ListObjects concurrency read limits ([#916](https://github.com/openfga/openfga/pull/916))
+
+  In our last release [v1.3.0](https://github.com/openfga/openfga/releases/tag/v1.3.0) we modified the default behavior of Check and ListObjects such that it limits/restricts the degree of concurrency that is allowed for a single request. This change was unintended. This release reverts the default behavior back to unbounded concurrency limits (the prior default). The change mostly affects those using OpenFGA as a library.
```

Comment:
Should we point them to how to set the values and what was the previous default value in 1.2.0?

Reply:
The limits didn't exist in `v1.2.0`, but they did in `v1.3.0`. Did you mean `v1.3.0`?

Reply:
@adriantam I'd like to point them here https://github.com/openfga/openfga.dev/pull/489

Can we merge that first and then I'll update this to point them there?